chingu-voyages/v18-bears-team-04
# Revert 8 ft login signup

#### Description
This PR create a user login and signup endpoints

#### Type of change
 create a user login and signup endpoint
- [x] login endpoint
- [x] signup endpoint

#### How Has This Been Tested?
- clone this repo
- checkout to this branch by running `npm checkout ft-login-signup`
- start the `Server` using `npm start`
-  goto postman and test the endpoint
- ensure you added .env files in your root folder


#### Checklist:

#### PT-ID
N/A
[](url)
N/A
#### Screenshots
![image](https://user-images.githubusercontent.com/33469121/78960786-02ba0180-7ae7-11ea-861d-cc32e3c76c0b.png)
![image](https://user-images.githubusercontent.com/33469121/78960804-0f3e5a00-7ae7-11ea-931a-2be7d6063e9b.png)


#### Questions:
